templates/style.css

```diff
@@ -0,0 +1,7 @@
+/*
+ * Theme Name: Whippet Theme
+ * Author: dxw
+ * Author URI: https://www.dxw.com/
+ * License: GPLv3
+ * License URI: http://www.gnu.org/licenses/gpl-3.0.txt
```

Comment:
Can we change the licence here? We're using MIT on this project: https://github.com/dxw/whippet-theme-template/blob/master/LICENSE.txt